code/modules/recycling/sortingmachinery.dm

```diff
@@ -277,18 +279,17 @@
 		for(var/obj/structure/bigDelivery/O in src)
 			deliveryCheck = 1
 			if(O.sortTag == 0)
-				O.sortTag = 1
+				O.sortTag = defaultDestinationTag
 		for(var/obj/item/smallDelivery/O in src)
 			deliveryCheck = 1
 			if(O.sortTag == 0)
-				O.sortTag = 1
+				O.sortTag = defaultDestinationTag
 		if(deliveryCheck == 0)
-			H.destinationTag = 1
+			H.destinationTag = defaultDestinationTag
 
 		sleep(10)
 		playsound(src, 'sound/machines/disposalflush.ogg', 50, 0, 0)
 		sleep(5) // wait for animation to finish
-
 		H.init(src)	// copy the contents of disposer to holder

```

Comment:
@Fox-McCloud It's part feature/fix in my opinion too. I can revert the code to be pure fix if you'd like.

To the other concern, it'll act similar to how it is now since the chutes create a `disposalholder` and dump it's contents into it (except now it's not creating n-1 holders that are just empty because n items bumped into it).

However I might bother with seeing if I can create a `disposalholder` for each item bumping in like I said earlier; making Cargo's life much more easier and not sending expensive items into the trash because a pop-can happened to enter first followed by the shipment.